src/Razor/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptorComparer.cs

```diff
@@ -59,7 +59,7 @@ public int GetHashCode(BoundAttributeDescriptor descriptor)
             var hash = HashCodeCombiner.Start();
             hash.Add(descriptor.Kind, StringComparer.Ordinal);
             hash.Add(descriptor.Name, StringComparer.Ordinal);
-            hash.Add(descriptor.IsEditorRequired);
+            hash.Add(descriptor.IsEditorRequired ? 1 : 0);
```

Comment:
I turned the `HashCodeCombiner.Add(object)` to `HashCodeCombiner.Add(T?)` here - https://github.com/dotnet/aspnetcore/pull/35208/files#diff-9dc07138e333898e4361271c6768a7be8a725cfa984f706a02029d576b67175aR41 which also fixes a couple of other places that box.

Reply:
💡 Consider porting `HashCode` from dotnet/runtime, e.g. how I implemented https://github.com/dotnet/roslyn-analyzers/pull/4863.

Reply:
⏱️ I'll pull this change out for now since it's only a small part of this PR and there is intent to implement it separately.

Reply:
➡️ Now removed